keyboards/kinesis/stapelberg_i2c/config.h

```diff
@@ -0,0 +1,37 @@
+#ifndef STAPELBERG_CONFIG_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```